.circleci/config.yml

```diff
@@ -0,0 +1,120 @@
+# .circleci/config.yml
+version: 2.1
+
+executors:
+  osx-arm64:
+    macos:
+      xcode: 14.2.0 # indicate your selected version of Xcode
+    resource_class: macos.m1.large.gen1
+  linux-aarch64:
+    machine:
+      image: ubuntu-2004:current
```

Comment:
It might be useful: Since recently it is possible to use Docker executor with ARM64 resources - https://circleci.com/docs/using-docker/#arm
This way you could use a custom Docker image that has most of the dependencies preinstalled. It will save a lot of time for the runs/builds!

Here is a small example:
- https://github.com/martin-g/circleci-arm64-test/blob/master/.circleci/config.yml#L17-L20
- https://app.circleci.com/pipelines/github/martin-g/circleci-arm64-test
Here I use `openEuler` Linux. CircleCI provides only Ubuntu images for the `machine` executor